Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/PlayerJoinedSessionMessage.java

```diff
@@ -21,6 +23,15 @@
     private final CarType carType;
     private final TeamColour team;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)